### ISSUE TITLE
Fix icons position, progress bar in tabBar

### ISSUE THREAD

**Issue by idAlencar** [minbrowser/min#1043]
hi @PalmerAL , I made some corrections in the tabBar.css 8e3d331b35726c8639b95ebc7ed349aafff76ebe,
Fixed the size of the icon and the progress bar to make better use of the spaces, avoiding lateral intrusions and occasional clicks, what do you think of that?

**Comment by PalmerAL** [comment]
Just to make it easier to see what's changed, I highlighted some of the buttons:
before:
<img width="1552" src="https://user-images.githubusercontent.com/10314059/83453126-8cb78400-a41f-11ea-8bc8-f9bf10498ff5.png">
after:
<img width="1552" src="https://user-images.githubusercontent.com/10314059/83453130-8fb27480-a41f-11ea-8f57-f0cb7954ffff.png">

* I like the fact that the new tab and new task buttons now have the same width.
* I don't think I agree with reducing the padding on the tabs - it looks strange when the are a lot of tabs open.
* I'm not sure I understand the reason for changing the back button - are there issues with the existing clickable area? Also, it seems like having the button in the middle of the clickable area makes more sense.

**Comment by idAlencar** [comment]
I will return the standard of the guides, after a few hours of use, I did not like the modification I made, I will just adjust the progress bar to match the same size of the guides.
As for the back button, I have a different view than yours on the mac (it looks very good), the manjaro looks different, so I made the changes.
tomorrow i do some pictures so we can compare

**Comment by idAlencar** [comment]
what do you think of the visual feedback of the button when clicking? I find it strange not to see an indication that the click occurred.
chrome uses a gray circle that appears only on the button icon when we click. I thought that when the click is made, the color of the icon can change quickly, a light gray, what do you think?

**Comment by PalmerAL** [comment]
I agree some visual feedback would be nice. Light gray icons are already used for disabled buttons (like the back button), so that might be confusing, but a circle like Chrome uses might work.

**Comment by idAlencar** [comment]
before:
see this rule of 3, the horizontal and vestical size are not symmetrical
1- Menu (35px) and back (33px) we have = 68px
2 - alignment 
![beforeregrade3](https://user-images.githubusercontent.com/58848807/83868105-b6fb9100-a700-11ea-9a7f-f38439ae32d1.png)

After:
We have menu (36px) and back (36px) = 72px
symmetrical height and width
2 - alignment correction in manjaro

![afterregrade3](https://user-images.githubusercontent.com/58848807/83868458-46a13f80-a701-11ea-9b30-5d6f7249231b.png)

I will reverse the modification in the tab space and test some effects on the buttons. weekend i give attention to Min
